examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c

```diff
@@ -768,200 +804,6 @@ ex: printf("Wrong parameters value: file %s on line %d\r\n", file, line) */
 }
 #endif
 
-static inline void print_user_config_debug_banner( void )

```

Comment:
awesome you were able to remove these lines and put all into common code - even if it isn't consumed from common file by the two projects 👍